src/redux/fallbackExplorer.js

```diff
@@ -475,10 +475,10 @@ export const fallbackExplorerInit = () => async (dispatch, getState) => {
       },
       type: FALLBACK_EXPLORER_SET_ASSETS,
     });
+    dispatch(explorerInitL2());
```

Comment:
polygon assets we're sometimes showing up so I moved this inside the mainnet check